assets/src/edit-story/components/keyboardShortcutsMenu/useKeyboardShortcuts.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useMemo } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import getKeyboardShortcuts from './getKeyboardShortcuts';
+
+function useKeyboardShortcuts() {
+  const keyboardShortcuts = useMemo(() => getKeyboardShortcuts(), []);
+  // Filter out disabled keyboard shortcuts for headers, panels, sections
+  const { headers, landmarks, sections } = useMemo(
+    () => ({
+      header: keyboardShortcuts.headers,
+      landmarks: keyboardShortcuts.landmarks.filter((o) => !o.disabled),
+      sections: keyboardShortcuts.sections.map((section) => ({
+        ...section,
+        commands: section.commands.filter((o) => !o.disabled),
+      })),
+    }),
+    [keyboardShortcuts]
+  );
+
+  return { headers, landmarks, sections };
+}
```

Comment:
Could we just statically define this instead of using a hook & memos to manipulate the static data? 

I know `isPlatformMacOS` relies on the navigator, but that's fine to read in a module scope as opposed to a component scope:
```suggestion
export const keyboardShortcuts = {
  header: getKeyboardShortcuts().headers,
  landmarks: keyboardShortcuts().landmarks.filter((o) => !o.disabled),
  sections: keyboardShortcuts().sections.map((section) => ({
    ...section,
    commands: section.commands.filter((o) => !o.disabled),
  })),
};
```

Reply:
Simplified it somewhat in 6449328c9. Thanks for the attention to detail!